UI/window-basic-settings-stream.cpp

```diff
@@ -1429,8 +1429,7 @@ bool OBSBasicSettings::ServiceSupportsCodecCheck()
 
 		cur_audio_name = ui->simpleOutStrAEncoder->itemText(
 			ui->simpleOutStrAEncoder->findData(cur_enc));
-		fb_audio_name = ui->simpleOutStrAEncoder->itemText(
-			ui->simpleOutStrAEncoder->findData(fb_enc));
+		fb_audio_name = (cur_enc == "opus") ? "AAC" : "Opus";
```

Comment:
Isn't fb_audio_name a displayed/translated string? Is there be a way to get the appropriate value without hard-coding like this? We do this in the `else` branch, though that also seems to rely on some hard-coding from `get_adv_audio_fallback()`:
```
fb_audio_name = obs_encoder_get_display_name(QT_TO_UTF8(fb_enc));
```

Reply:
Old Message:
> I answer by e-mail so it might not be in the related conversation.
>
>In Simple mode, only codec name are used (Opus, AAC) and are not sourced from translations. Those name are harcoded.
>The user choose a codec not a specific implementation.
>
>In Advanced mode, encoder implementation names are used since the user can choose the implementation in this mode.

New message:
I made audio codec names shown in the UI translatable.

> though that also seems to rely on some hard-coding from `get_adv_audio_fallback()`

Like the video one.

Reply:
> Is there be a way to get the appropriate value without hard-coding like this?

In simple mode, no. If AAC is not available we need to switch to Opus and vice versa. There is `get_simple_fallback` for video which is also hardcoded.

In advanced mode,

Reply:
Not sure if it's really necessary to make the codec name translateable since as proper nouns they shouldn't be translated.